### PR TITLE
block/text-with-image/styles: adjusting styles to latest changes

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: apps/home/templates/apps_home/blocks/text_with_image_block.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 {% image value.image width-500 as image %}
 
-<div class="{% if self.background_color %}block__{{ self.background_color }}{% else %}block__body-bg{% endif %}">
+<div class="{% if self.background_color %}ds-block__{{ self.background_color }}{% else %}block__body-bg{% endif %}">
     <div class="container-fluid gx-0">
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html:5]: hmm yeah gx-0 is good to avoice where possible as can mess things up on smaller screens but it works here, what I might do is as we decvelope check if we ever use the full width container-fluid without gx-0 (as doesn't seem useful to have gutters on full width element) and if not we set that as a default in base.scss but we can check how it goes, here I think it's fine.

[the remark this answers is not in the document]
Comment (khamui) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html:5]: okay that makes sense!

[the remark this answers is not in the document]
Comment (khamui) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html:5]: i am leaving for now, there are already location using `container-fluid`

File: apps/home/templates/apps_home/blocks/text_with_image_block.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-        <div class="row">
+        <div class="row align-items-center">
             <div class="col-12 col-sm-4">
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/text_with_image_block.html]: we can totally use 2 break points, we normally don't have tablet size designs as we don't have alot of people accessing our sites on middle size screens so I normally just check it and maybe add an extra col size where appropriate

File: digitalstrategie/assets/scss/components/_block.scss [hunk 3]
```diff
@@ -105,13 +105,18 @@
...
 
 // this helps to align text symmetrical on mobile,
 // but kind of leftish if viewed on bigger screen
 .textimg-block-text__wrapper {
     width: 100%;
-    padding: 0 1*$spacer;
+    padding: 0 1 * $spacer;
+
+    & h1 {
+        margin: 1 * $spacer 0;
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply digitalstrategie/assets/scss/components/_block.scss:118]: this can be deleted now you updated the tmeplate right?

**Pull request merged**